content/internships/contents.lr

```diff
@@ -43,15 +43,64 @@ If you're interested in this program, please review the resources below and come
 
 Here are our mentors for this round of internships:
 
-| Name                          | Slack Username          | About                                                                            |
-| ----------------------------- | ----------------------- | -------------------------------------------------------------------------------- |
-| Alden Page                    | `@aldenpage`            | [See CC Open Source Blog author page](/blog/authors/aldenpage/)                  |                  
-| Anna Tumadóttir 	            | `@Anna`                 | [See CC staff bio](https://creativecommons.org/author/annacreativecommons-org/)  |
-| Ari Madian 	                  | `@Ari` 	                | [See CC Open Source Blog author page](/blog/authors/akmadian/)                   |
-| Brent Moran 	                | `@Brent`                |	[See CC Open Source Blog author page](/blog/authors/mathemancer/)                |
-| Dhruv Bhanushali 	            | `@dhruvkb` 	            | [See CC Open Source Blog author page](/blog/authors/dhruvkb/)                    |
-| Hugo Solar 	                  | `@hugo`                 |	[See CC Open Source Blog author page](/blog/authors/hugosolar/)                  |
-| Kriti Godey 	                | `@kriti` 	              | [See CC Open Source Blog author page](/blog/authors/kgodey/)                     |
-| María Belén Guaranda Cabezas 	| `@Maria Belen Guaranda` |	[See CC Open Source Blog author page](/blog/authors/soccerdroid/)                |
-| Sarah Hinchcliff Pearson 	    | `@shinchpearson`        |	[See CC staff bio](https://creativecommons.org/author/sarahcreativecommons-org/) |
-| Timid Robot Zehta 	          | `@Timid Robot`          |	[See CC Open Source Blog author page](/blog/authors/TimidRobot/)                 |
+<div class="table-container padding-vertical-normal">
```

Comment:
It didn't work if you surround the markdown format with the `.table-container` class?